src/client-rest/middleware.ts

```diff
@@ -427,9 +427,10 @@ export async function getApiProductTables(
   const promises = tableFileNames.map((f) => readJsonFile<ProductTableFile>(filesDir, f));
   const tableFilesContent: ReadonlyArray<ProductTableFile> = await Promise.all(promises);
   // Map to the API objects to return
+  const childFiles: LoadedFiles = {};
```

Comment:
I think we can declare this as `Record<string, string>` as it will be mutable in all use cases.